packages/test/stochastic-test-utils/src/describeFuzz.ts

```diff
@@ -3,6 +3,8 @@
  * Licensed under the MIT License.
  */
 
+import process from "process";
+
 function createFuzzSuite(
 	tests: (this: Mocha.Suite, args: FuzzSuiteArguments) => void,
```

Comment:
Changes in this file were required to use 'stochastic-test-utils' in the browser.

I configure webpack to generate an empty module for 'process'.
As a result, "process.env" is undefined, hence the '?.' below.